src/org-manager/components/back-link/back-link.component.ts

```diff
@@ -10,10 +11,15 @@ import * as fromOrganisationPendingStore from '../../../org-manager/store';
 export class BackLinkComponent {
   @Input() public store: Store<fromOrganisationPendingStore.OrganisationRootState>;
 
-  constructor() { }
+  constructor(private route: ActivatedRoute) { }
 
   public onGoBack() {
-    this.store.dispatch(new fromRoot.Back());
+    this.route.url.subscribe(url => {
```

Comment:
this may leave a number of subscriptions laying around that aren't used. Better to save the reference to the subscription, so we can unsubscribe in the onDestroy method

Reply:
@andywilkinshmcts  The function will be called justo onGoBack, when back button is clicked.

@ronaldmansveld  I am open to suggestions, I need to check the current URL, if it is 
'active-organisation' back link should go to '/pending-organisations' , in all other cases it should go to previous visited page.